src/site/content/en/blog/logical-property-shorthands/index.md

```diff
@@ -0,0 +1,450 @@
+an image with a caption to handle different writing and document directions.
+
+<figure class="w-figure w-figure--fullbleed">
+  <video playsinline controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/atoms-sandbox.google.com.a.appspot.com/logical-property-figure-demo.mp4">
+  </video>
+</figure> 
+
+Or try it!
+
+{% Glitch 'logical-figure' %}
+
+You don't have to do much to make a card internationally responsive with a 
+`<figure>` and a few logical properties. If you're curious how all this internationally 
+considerate CSS works together, I hope this is a small meaningful introduction.
+
+### Polyfilling & Cross Browser Support
+The Cascade or build tools are viable options to have old and new browsers alike, 
+properly spaced with updated logical properties. By following a physical property 
+with a logical one, the browser will use the "last" one it's found during style 
+resolution. Here's a polyfill by hand:
+
+```css
+p {
+  /* for unsupporting browsers */
+  margin-top: 1ch;
+  margin-bottom: 2ch;
+  
+  /* for supporting browsers to use */
+  /* and unsupporting browsers to ignore and go 🤷‍♂️ */
+  margin-block: 1ch 2ch;
+}
+```
+
+You could also use `@supports` to determine whether or not to provide physical 
+property fallbacks:
+
+```css
+p {
+  margin-top: 1ch; 
+  margin-bottom: 2ch;
+}
+
+@supports (margin-block: 0) {
+  p {
+    margin-block: 1ch 2ch;
+  }
+}
+```
+
+[Sass](https://sass-lang.com), [PostCSS](https://github.com/csstools/postcss-logical), 
+[Emotion](https://emotion.sh) and others have automated bundler and/or build 
+time offerings that have a wide array of fallbacks or solutions. Check out each one 
+for it's strategy to see which matches your toolchain and overall site strategy.
+
+## What's Next
+More of CSS will offer logical properties, it's not done yet! There's 1 big missing 
+set of shorthands though, and a resolution is still pending in this [Github issue](https://github.com/w3c/csswg-drafts/issues/1282). 
+There is a temporary solution [in a draft](https://drafts.csswg.org/css-logical/#logical-shorthand-keyword). What if you want to style all 
+logical sides of a box with a shorthand?
+
+<div class="w-columns">
+{% Compare 'better', 'Physical Shorthand' %}
+```css
+margin: 1px 2px 3px 4px;
+margin: 1px 2px;
+margin: 2px;
+```
+{% endCompare %}
+
+{% Compare 'better', 'Logical Shorthand' %}
+```css
+margin: logical 1px 2px 3px 4px;
+margin: logical 1px 2px;
+margin: logical 2px;
+```
+{% endCompare %}
+</div>
+
+The current draft proposal would mean you have to write `logical` in every 
+shorthand in order to get the logical equivalent applied, which doesn't sound 
+very D.R.Y. to some. There are other proposals to change it at the block or page level, 
+but that could leak logical uses into styles still assuming physical sides. 
+It's a tough one! Cast your vote, voice your opinion, we want to hear from you.
```

Comment:
Is it worth being explicit here how they do that? (comment on the issue).

Reply:
i think so! let me see if adding a couple small snippets can bring some clarity here 👍  otherwise, a comment in that thread is meaningful too